### DISCUSSION TITLE
biplane localization

### DISCUSSION THREAD

**Discussion by Remyt2** [jungmannlab/picasso#329] [Q&A]
Hi everyone.  I've recently started using a Bruker Vutara VXL SR.  I expressed interest in using Picasso to our FAS and this was the response...

_I haven't used Picasso myself yet, can it do biplane localization?
If not, you probably won't be able to get z information out of SR collected images with Picasso._

This is Bruker's description of their biplane tech...

_Vutara specializes in three-demensional (3D) super-resolution SML microscopy by utilizing our patented biplane imaging approach (see Juette et al., Nat. Meth. 5: 527-9 (2009), and Mlodzianoski, et al., Opt. Express 17(10):8264 (2009)).  Briefly, this technique is based on imaging of a photoswitchable/ photoactivatable sample at two focal planes simultaneously.  Optically, this is accomplished by splitting the detection path of a conventional widefield fluorescence microscope into two beams of slightly varying optical length.  By comparing the imaged characteristics of a given single molecule emitter across the two focal planes, the emitter’s location can be computationally inferred with sub-diffraction precision in all three dimensions._  

I can export raw data as a tiff stack, but can Picasso do biplane localization?

Thanks for help with this.

Remy

**Comment by rafalkowalewski1** [comment]
Hey Remy, I'm sorry but Picasso does not allow for this. The 3D localization technique we use is based on an astygmatic lens. I hope this is of some help.

Bests,
Rafal

**Comment by Remyt2** [comment]
Thanks for the response, Rafal.  Although out of the scope of this environment as a Picasso forum, can biplane localization allow for accurate molecule counting with qPAINT, or should I be looking for another instrument that uses astygmatic lenses like ONI's nanoimager?

Thanks again

Remy

**Comment by rafalkowalewski1** [comment]
Hi Remy, I think that qPAINT should be compatible with biplane localization, although our lab has never used it. 

Bests,
Rafal
